src/Microsoft.DotNet.Build.Tasks/PackageFiles/init-tools.cmd

```diff
@@ -0,0 +1,33 @@
+@echo off
+setlocal
+
+set PROJECT_DIR=%~1
+set TOOLS_DIR=%~2
+set DOTNET_CMD=%~3
+
+echo PROJECT_DIR=[%PROJECT_DIR%]
+echo TOOLS_DIR=[%TOOLS_DIR%]
+echo DOTNET_CMD=[%DOTNET_CMD%]
+
+if not exist "%PROJECT_DIR%" (
+  echo ERROR: Cannot find project root path at [%PROJECT_DIR%]. Please pass in the source directory as the 1st parameter.
+  exit /b 1
+)
+
+if not exist "%TOOLS_DIR%" (
+  echo ERROR: Cannot find tools path at [%TOOLS_DIR%]. Please pass in the tools directory as the 2nd parameter.
+  exit /b 1
+)
+
+if not exist "%DOTNET_CMD%" (
+  echo ERROR: Cannot find dotnet command at [%DOTNET_CMD%]. Please pass in the dotnet command as the 3rd parameter.
+  exit /b 1
+)
+
+set TOOLRUNTIME_DIR=%PROJECT_DIR%\bin\obj\Windows_NT.AnyCPU.Debug\ToolRuntime
```

Comment:
We are likely going to need to change this location because if someone does a clean we don't want to be try and delete the tools we are running on as that will fail badly.